test/skimNtuple2016_HHbtag.cpp

```diff
@@ -4673,6 +4701,7 @@ int main (int argc, char** argv)
 
         // build the tlv
         TLorentzVector tlv_additionalJet(theBigTree.jets_px->at(iJet), theBigTree.jets_py->at(iJet), theBigTree.jets_pz->at(iJet), theBigTree.jets_e->at(iJet));
+        if (doSmearing) tlv_additionalJet = Smearer.getSmearedJet(tlv_additionalJet, theBigTree);
```

Comment:
Can you double check if this is the only point where we should shift additional jets? I think so but I'm not 100% sure.